zkb

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 # Files that refer to this file by ID. (Backlinks)
 
+ZKPATH="${ZKPATH:-~/Dropbox/wiki/archive}"
```

Comment:
This will use `$ZKPATH` if previously exported in your shell, or default to the value after the `:-`. This worked for me, but I only tested in zsh. What shell do you use?

If you're game for leaning into `$ZKPATH`, we could bail early if not set.

Reply:
I'm using fish, but I tested by just running bash (inside fish, I'm sure there's some complications there that I don't fully understand)

Yeah, let's just assume a `$ZKPATH`, and we can always upgrade from there later if we continue. I'm mostly making that call on a "simpler is better, and there's just the two of us right now" kind of way.